packages/fxa-profile-server/bin/server.js

```diff
@@ -5,10 +5,6 @@
 // Important! Must be required first to get proper hooks in place.
 require('../lib/monitoring');
 
-// Only `require()` the newrelic module if explicity enabled.
-// If required, modules will be instrumented.
-require('../lib/newrelic')();
```

Comment:
Did you mean to include these changes?

Reply:
Yea, I noticed this wasn't being used and seemed alright to include since wasn't being used. I can take them out if you like.